packages/documentation/src/pages/getting-started/common-tasks/new-unit-test.mdx

```diff
@@ -4,23 +4,21 @@ tags: Mocha, Chai, Enzyme, schemaform-utils
 ---
 
 # Writing a unit test
-
 All new code that is added to `vets-website` should be unit tested and unit tests should cover at least 75% of code paths. Write unit tests as you build to make sure your form (or other component) is behaving as you expect and to help guard against future bugs.
-
 For example, you might create a unit test file for each page in a form and then test the following scenarios:
-
 - The correct number of inputs show up when you render the page.
 - The correct number of fields display validation errors if you submit without entering any information.
 - Any conditional logic on the page displays under the correct conditions.
 
 ## Unit test overview
-
 - The `vets-website` repo uses:
   - [Mocha](https://mochajs.org/) for running unit tests
   - [Chai](http://chaijs.com/) for test assertions
-  - [Enzyme](https://airbnb.io/enzyme/) for mounting and inspecting React components
+  - [React Testing Library](https://testing-library.com/docs/react-testing-library/intro) and [Enzyme](https://airbnb.io/enzyme/) for mounting and inspecting React components
```

Comment:
If you refer to it as "React Testing Library (RTL)" here, I think it should be fine to refer to it as RTL later in the doc.